uber/tests/models/test_job.py

```diff
@@ -46,7 +46,8 @@ def test_non_volunteer(self, session):
 
     def test_restricted(self, session):
         assert session.assign(session.staff_two.id, session.job_six.id)
-        session.staff_two.trusted = True
+        session.staff_two.trusted_depts = str(c.CONSOLE)
+        # TODO: update
         session.commit()
         assert not session.assign(session.staff_two.id, session.job_six.id)
 
```

Comment:
@eli I am getting confused by this test, it appears to imply that session.job_six is a restricted job, but I don't see that being set anywhere.  I didn't see any text fixtures that appear to be setting it either.  I would need to set job_six.location = c.CONSOLE i guess?  need to work on this.